### PR TITLE
WIP Add proper names to markdownlint.json

### PR DESCRIPTION
**Pull request by Youssef1313**
Inspired from AspNetCore.Docs config.
Currently, there was a recent PR that added nuget in incorrect casing.
The build of the first commit must fail. After I confirm that, I'll fix the casing.

TODO: Add GitHub to proper names too.

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -39,5 +39,15 @@
     "MD041": false,
     "MD046": {
         "style": "fenced"
+    },
+    "proper-names": {
+        "code_blocks": false,
+        "names": [
+            ".NET",
```
Comment (Youssef1313) [inline .markdownlint.json:46]: There are too many .NET casing violations. Do you want to fix them or disable?

File: .markdownlint.json [hunk 2]
```diff
@@ -39,5 +39,17 @@
...
+            "ASP.NET",
+            "JavaScript",
+            "NuGet",
+            "nuget.org",
+            "NuGet.org",
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply .markdownlint.json:50]: ping @BethMassi What is the correct use here?

[the remark this answers is not in the document]
Comment (BethMassi) [reply .markdownlint.json:50]: NuGet is the product name. When referring to the website, either are fine as that is not the product name. Just like .NET is the product name but the website is dot.net.

[the remark this answers is not in the document]
Comment (BillWagner) [reply .markdownlint.json:50]: Thanks @BethMassi

**Review by BillWagner — DISMISSED** [review]
This LGTM @Youssef1313 

I'll :shipit:

**Comment by Youssef1313** [comment]
@BillWagner this is not ready yet. I forgot to add WIP 😄

**Comment by BillWagner** [comment]
> this is not ready yet. I forgot to add WIP 😄

Instead, can you convert it to a `draft` PR?

**Comment by Youssef1313** [comment]
> 
> 
> > this is not ready yet. I forgot to add WIP 😄
> 
> Instead, can you convert it to a `draft` PR?

Sure

**Comment by Youssef1313** [comment]
@BillWagner Can you review the latest commits please?

File: docs/core/tools/dotnet-nuget-add-source.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: dotnet nuget add source command
 description: The dotnet nuget add source command adds a new package source to your NuGet configuration files. 
 ms.date: 03/20/2020
 ---
-# dotnet nuget add source
+# `dotnet nuget add source`
```
Comment (adegeo) [inline docs/core/tools/dotnet-nuget-add-source.md:6]: I recently saw these `blocks` removed from headers in another PR, as if we're not supposed to have them.

Comment (tdykstra) [reply docs/core/tools/dotnet-nuget-add-source.md:6]: Right, we don't apply styles to headings.  https://docs.microsoft.com/en-us/contribute/text-formatting-guidelines#headings-and-link-text

**Review by tdykstra — COMMENTED** [review]
A couple of comments on text formatting -- per style guidelines, file names should be italicized, not code style, and text styles shouldn't be applied to headings.

**Review by CamSoper — APPROVED** [review]
Whoops, accidental approval.

**Review by CamSoper — CHANGES_REQUESTED** [review]
Take a pull from `master` and it should resolve any build warnings coming from the Azure SDK package list. No merging necessary, take the one in `master` as-is.

**Comment by adegeo** [comment]
@Youssef1313 We're closing older PRs that have gone stale. Feel free to reopen if you're able to address the issues raised.  Cheers!

**Pull request closed without merging**